### ISSUE TITLE
make TOC stick to the visible viewport

### ISSUE THREAD

**Issue by martenson** [galaxyproject/galaxy-hub#103]
So it does not disappear when you scroll down.

**Comment by dannon** [comment]
@martenson Good catch, meant to do this when I set that up but it fell through the cracks.  Done in #104

**Comment by martenson** [comment]
Seems to have some unwanted side effects. E.g. at https://new.galaxyproject.org/public-galaxy-servers/

![screenshot 2017-02-02 13 54 50](https://cloud.githubusercontent.com/assets/1814954/22563603/3ec20254-e94f-11e6-8075-a92b5afbe8ef.png)

**Comment by dannon** [comment]
Pfft, good catch.  That's sure interesting, it looked great on the smaller nav pages!  I'll tinker with the affix to make it work.

**Comment by dannon** [comment]
A big step forward in rendering subsections in 69fdfe18a5fb17607f96f1943945ab5a16e5f8e6, but I need to make it scrolly too.  Pure affix won't work here, for *very* long lists like the galaxy-public-servers.

**Comment by martenson** [comment]
Given the https://new.galaxyproject.org/public-galaxy-servers/ has now very different structure maybe the number of important pages that this overlapping would affect is low enough to go ahead.
